pkg/asset/cluster/cluster.go

```diff
@@ -102,3 +103,13 @@ func (c *Cluster) Generate(parents asset.Parents) error {
 func (c *Cluster) Files() []*asset.File {
 	return []*asset.File{c.file}
 }
+
+// Load returns error if the tfstate file is already on-disk, because we want to
```

Comment:
I am having a hard time understanding this comment. If the terraform state file is already on-disk, why would that re-launch the cluster? The cluster is launched via `Generate`, which would not be called if the file is found.

Reply:
Just want to error out in the case here.